src/default_config.yaml

```diff
@@ -18,28 +18,34 @@ chia_logs:
     remote_user: "chia"
     remote_port: 22
 
+# All services and thus handlers are enabled by default
+monitored_services:
+  - FULL_NODE
+  - HARVESTER
+  - FARMER
+  - WALLET
```

Comment:
I think this should go in the example config for users, or are you keeping it for a follow-up PR?

Reply:
If this would be omitted, no handlers would get init by default. That's certainly a stance we can take, just not sure we're on the same page about the effects?

Reply:
Oh now I get what you meant. Added also to example with more guidance.